docs/StandardLibrary.md

```diff
@@ -0,0 +1,334 @@
+# Flux Standard Library Design
+
+The Flux standard library will define the set of functions, constants, and other
+values that are available out of the box with Flux, providing the functionality that is
+most frequently used.
+
+Some functions in the standard library will be available without qualifying
+them with a namespace.  These will be the most common functions, to help Flux
+code be more concise.  Such functions are said to be in the "top level package."
+
+The remaining functions in the standard library will be divided up into
+discrete packages that are organized by topic.
+
+What's presented here is just a departure point for further discussion.
+
+## The Top Level Package
+
+Those functions that provide functionality that is very frequently
+used and should not need to be qualified with a namespace in Flux code.
+(Consider also the concept of "prelude")
```

Comment:
The more I think about it the more I like the concept of "prelude", it helps to remove lots of edge cases.

Reply:
I missed the all hands in September, so I'm not that familiar with the idea.  The "prelude" would be an option that is provided to the compiler, that is a list of packages that do not need namespace qualification?